BriefAssistant/Models/CircuitCourtCase.cs

```diff
@@ -186,5 +190,8 @@ public class CircuitCourtCase
         [Required]
         [DataMember]
         public string OpponentName { get; set; }
+        [Required]
```

Comment:
This should not be be marked as Required. By marking it as required, the brief controller will expect it to be in the request, and will throw an exception when its not there